helm/solr-operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart Solr Operator
+name: solr-operator
+version: 0.1.0
```

Comment:
Should this match the version of the operator? Not sure how this is generally used.

Reply:
If you're pointing to latest, how would we determine the matching version?

Reply:
Seems arbritrary? Here's a link on version & appVersion:
https://codefresh.io/docs/docs/new-helm/helm-best-practices/#chart-versions-and-appversions

Reply:
Generally, I've incremented `version` with each Helm chart change, `appVersion` is optional, perhaps we leave it out?

Reply:
Yeah, we won't change the inputs to the solr operator very often I think. So keeping this as 0.1.0 is probably fine.